apps/expo/src/components/ResetPassword.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react";
+import { Text, TextInput, TouchableOpacity, View } from "react-native";
+import Toast from "react-native-root-toast";
+import { useSignIn } from "@clerk/clerk-expo";
+
+interface IClerkError {
+  status: number;
+  clerkError: boolean;
+  errors: {
+    code: string;
+    message: string;
+    longMessage: string;
+    meta: {
+      paramName: string;
+    };
+  }[];
+}
```

Comment:
why is this interface being created manually? can it not be inferred / imported?

Reply:
when you catch an error the type is `unknown`, and to be able to access fields on it you must cast it to some known type. Clerk actually doesn't export this, so we have to manually create it